### ISSUE TITLE
Balasana: aria-expanded accessibility property does not work on mobile or desktop menu

### ISSUE THREAD

**Issue by jp-imagines** [Automattic/wp-calypso#58471]
### Quick summary

`aria-expanded` is used to indicate the state of an expandable element. In the _Balasana_ theme, this option remains `false`, even when a parent menu item is expanded (with a keyboard or screen reader).

### Steps to reproduce

1. Activate the Balasana theme.
2. Ensure you have at least one "parent" menu item with submenu "child" items.
3. Visit your site, and use the browser inspector to find the `aria-expanded` element on the parent menu item.
4. Use the Tab key to "hover" over the parent menu item without a mouse.

### What you expected to happen

`aria-expanded` should change from `false` to `true`.

### What actually happened

It never changes from `false`.

### Context

32899066-hc

### Theme-specific issue?

Balasana theme.

### Reproducibility

Consistent

### Severity

Some (< 50%)

### Available workarounds?

No but the platform is still usable

**Comment by kosiew** [comment]
Hi @jp-imagines ,

Thanks for reporting this and the detailed steps.

I could reproduce this on a Simple site.

Transferring to themes repo for developers' further investigation.
